.github/workflows/fqdn-perf.yaml

```diff
@@ -240,3 +278,15 @@ jobs:
           results_bucket: ${{ env.GCP_PERF_RESULTS_BUCKET }}
           artifacts: ./perf-tests/clusterloader2/report/*
           other_files: cilium-sysdump-final.zip ./perf-tests/clusterloader2/cl2-output.txt
+
+  commit-status-final:
+    if: ${{ always() && github.event_name != 'push' }}
```

Comment:
Small change, since there is no trigger for `push` we do not need this part of the if condition as well `&& github.event_name != 'push'`

Reply:
Thanks for noticing, fixed.